x-pack/test/functional/apps/discover/group1/reporting.ts

```diff
@@ -141,15 +146,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       it('is available if new', async () => {
         await reporting.openExportPopover();
-        expect(await reporting.isGenerateReportButtonDisabled()).to.be(null);
-        await exports.closeExportFlyout();
+        expect(await exports.isPopoverItemEnabled('CSV')).to.be(true);
+        await reporting.openExportPopover();
```

Comment:
Shouldn't it be `await exports.closeExportFlyout();`?
Same in other modified tests.

Reply:
Hey @jughosta! I know, it's a bit unintuitive. The `reporting.openExportPopover()` doesn't open the export flyout, but the export popover. Some tests were failing because that popover might remain open/reopen after generating an export from the flyout, resulting in some elements below it being unclickable. Since we don't have a close button, I used the same strategy as in other fn tests in Kibana which is clicking again on the button that opens the popover 🙂